### ISSUE TITLE
Are there any shortcuts to control video in Chrome?

### ISSUE THREAD

**Issue by wonseukk** [crittermike/shortkeys#632]
I have a job reviewing short videos. This is a must-have shortcut for me.

**Comment by AlexByte** [comment]
you can use another extension btw

**Comment by crittermike** [comment]
Yes! Shortkeys v5 has extensive video control shortcuts built in. There are dedicated actions for: play/pause video, mute/unmute video, increase/decrease playback speed, skip forward/backward, toggle fullscreen, toggle picture-in-picture, and more. You can find them in the 'Video' category when selecting a behavior in the options page.
